src/unitxt/standard.py

```diff
@@ -48,14 +50,37 @@ def verify(self):
                 )
             if self.demos_pool_size < self.num_demos:
                 raise ValueError(
-                    f"demos_pool_size must be bigger than num_demos={self.num_demos}, Got demos_pool_size={self.demos_pool_size}"
+                    f"demos_pool_size must be bigger than num_demos ({self.num_demos}), Got demos_pool_size={self.demos_pool_size}"
+                )
+            if self.loader_limit and self.demos_pool_size > self.loader_limit:
```

Comment:
I thin it should be >=. We don't want all the examples to go into the demo pool either, right?

Reply:
Not clear - if you don't do fine/prompt tuning it's not a problem to take all training to the demo pool.